website/src/app/kb/user-guides/windows-client/readme.mdx

```diff
@@ -106,6 +106,31 @@ To export or clear your logs:
 
 ## Troubleshooting
 
```

Comment:
Should I add a "how to set the log filter to debug" section too? I already wrote up the steps, just need to make them user-friendly.